web-app/src/pages/SignIn/ForgotPassword.tsx

```diff
@@ -0,0 +1,61 @@
+import { gql, useMutation } from "@apollo/client";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { getErrorMessage } from "../../utils";
+import { HOME_PATH } from "../paths";
+import "react-toastify/dist/ReactToastify.css";
+
+const SEND_RESET_PASSWORD_EMAIL = gql`
+  mutation SendResetPasswordEmail($email: String!) {
+    sendResetPasswordEmail(email: $email)
+  }
+`;
+
+const ForgotPassword = () => {
+  const navigate = useNavigate();
+  const [email, setemail] = useState("");
+  const [sendResetPasswordEmail] = useMutation(SEND_RESET_PASSWORD_EMAIL);
```

Comment:
I think you forget the types here : 
`  const [sendResetPasswordEmail] = useMutation<
    SendResetPasswordEmailMutation,
    SendResetPasswordEmailMutation
  >(SEND_RESET_PASSWORD_EMAIL);`

Reply:
ok